Lib/test/test_capi/test_float.py

```diff
@@ -197,6 +198,9 @@ def test_pack_unpack_roundtrip_for_nans(self):
                     # PyFloat_Pack/Unpack*() API.  See also gh-130317 and
                     # e.g. https://developercommunity.visualstudio.com/t/155064
                     signaling = 0
+                    if platform.machine().startswith('parisc'):
+                        # HP PA RISC uses 0 for quiet
```

Comment:
Maybe you can provide some reference.  E.g. this documented also in https://en.wikipedia.org/wiki/NaN#Encoding.